sphinxcontrib/datatemplates/directive.py

```diff
@@ -71,7 +72,7 @@ def run(self):
         app = env.app
         builder = app.builder
 
-        data_source = self.options['source']
+        data_source = self.arguments[0]
```

Comment:
As written, this is going to be a breaking change. How about if we deprecate the source option, remove the docs, and only use self.arguments[0] if self.options['source'] isn't there or is empty?

Reply:
Yes, that would be the better solution.